gcp/api/run_tests.sh

```diff
@@ -18,12 +18,13 @@ if [ $# -lt 1 ]; then
   exit 1
 fi
 
-pipenv requirements > requirements.txt
-virtualenv ENV
-. ENV/bin/activate
-pip install -r requirements.txt
+# pipenv requirements > requirements.txt
+# virtualenv ENV
+# . ENV/bin/activate
+# pip install -r requirements.txt
+python3 -m pipenv sync
```

Comment:
nit: Want to delete the commented lines?
Any idea why it used virtualenv instead of pipenv in the first place?

Reply:
Not sure why, I asked Oliver a few weeks ago and he can't remember exactly why either, I'll remove it.